app/src/main/java/org/schabi/newpipe/local/subscription/ImportConfirmationDialog.java

```diff
@@ -53,8 +85,8 @@ public Dialog onCreateDialog(@Nullable final Bundle savedInstanceState) {
     public void onCreate(@Nullable final Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
-        if (resultServiceIntent == null) {
-            throw new IllegalStateException("Result intent is null");
+        if (mode == 0 && value == null && serviceId == 0) {
+            throw new IllegalStateException("Input data not provided");
         }
```

Comment:
This seems to crash when rotating the screen while the confirmation dialog is visible

Reply:
At that point the module is not yet restored, I don’t know what this is supposed to do?

Reply:
Should be fixed in the commit I added, I left out the check since it will always use the `@Parcelize` decoders from the `SubscriptionImportInput` subclasses.